app.js

```
@@ -20,28 +24,28 @@ app.use(express.static(path.join(__dirname, "public")));
-        uniquesocket.emit("playerRole", "b");
+        players.black = socket.id;
+        socket.emit("playerRole", "b");
     } else {
-        uniquesocket.emit("spectatorRole");
+        socket.emit("spectatorRole");
     }
 
-    uniquesocket.on("disconnect", function () {
+    socket.on("disconnect", function () {
         if (uniquesocket.id === players.white) {
             delete players.white;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)